pkg/reconciler/v1alpha1/pipelinerun/pipelinerun_test.go

```diff
@@ -362,6 +360,8 @@ func TestReconcile_InvalidPipelineRuns(t *testing.T) {
 		})
 	}
 }
+
+/*
 func TestReconcile_InvalidPipelineRunNames(t *testing.T) {
```

Comment:
This is the only test that I can't figure yet how to "migrate" as it uses the `logs` from testAssets that aren't available (as a result of the controller change). I wonder if there is a better way to test this without having to "parse" the logs